src/UE4SSProgram.cpp

```diff
@@ -750,11 +756,18 @@ namespace RC
     {
         Output::send(STR("Setting up mods...\n"));
 
-        if (!std::filesystem::exists(m_mods_directory))
+        if (!std::filesystem::exists(m_mods_directory)) { set_error("Mods directory doesn't exist, please create it: <%S>", m_mods_directory.c_str()); }
+
+        auto cpp_sdk_path = m_mods_directory / "cppsdk.dll";
```

Comment:
Related to [570c637](https://github.com/UE4SS-RE/RE-UE4SS/pull/101/commits/570c637e2bd46884537ddba0221ca25a8b865342).
This should be renamed to `UE4SS-cppsdk.dll`.